README.rst

```diff
@@ -32,6 +32,7 @@ The required dependencies to use the software are:
 * SciPy >= 0.7
 * Scikit-learn >= 0.12.1
 * Nibabel >= 1.1.0.
+* Six
```

Comment:
I was thinking we could use the 'six' version that is shipped in sklearn.externals.